db/schema.rb

```diff
@@ -172,10 +172,10 @@
     t.text     "prediction_feedback"
     t.boolean  "is_hidden",                        :default => false
     t.text     "hint"
-    t.boolean  "is_full_width",            :default => false
-    t.boolean  "show_in_featured_question_report", :default => true
+    t.boolean  "is_full_width",                    :default => false
     t.integer  "interactive_id"
     t.string   "interactive_type"
+    t.boolean  "show_in_featured_question_report", :default => true
```

Comment:
There is frequently weird churn in `schema.rb` where the changes are simply whitespace, or the order in which fields appear.  I am not sure what we can do about it.  Usually I review the changes, and if its whitespace only, I just replace my `schema.rb` with  whatever is checked in.

Reply:
I do the same. I usually try not to commit schema.rb with whitespace changes only. But here I had so many files to commit that I didn't notice it probably.